tck/src/main/java/ee/jakarta/tck/data/standalone/entity/EntityTests.java

```diff
@@ -394,8 +394,16 @@ public void testBeyondFinalPage() {
         assertEquals(0, page.stream().count());
         assertEquals(false, page.hasContent());
         assertEquals(false, page.iterator().hasNext());
-        assertEquals(43L, page.totalElements());
-        assertEquals(5L, page.totalPages());
+        try {
+            assertEquals(43L, page.totalElements());
+            assertEquals(5L, page.totalPages());
+        } catch (UnsupportedOperationException x) {
+            // Some NoSQL databases lack the ability to count the total results
+            // and therefore cannot support a return type of Page.
+            // Column and Key-Value databases might not be capable of sorting.
+            // Key-Value databases might not be capable of Between.
+            return;
```

Comment:
Don't allow relational databases to raise UnsupportedOperationException,
```suggestion
            if (type.isKeywordSupportAtOrBelow(DatabaseType.GRAPH)) {
                // Some NoSQL databases lack the ability to count the total results
            } else {
                throw x;
            }
```